metricbeat/module/couchbase/syncgateway/_meta/docs.asciidoc

```diff
@@ -0,0 +1,23 @@
+Sync Gateway is the synchronization server in a Couchbase for Mobile and Edge deployment. This metricset allows to monitor a Sync Gateway instance by using its REST API.
+
+Sync Gateway access `[host]:[port]/_expvar` on Sync Gateway nodes to fetch metrics data, ensure that the URL is accessible from the host where Metricbeat is running.
+
+[float]
+=== Skipping some events
+
+As mentioned before, this metricset generates 4 types of events under `couchbase.syncgateway.*` JSON path: DB stats, global stats, Memory stats and per replication stats. Only DB stats are always sent but you can choose to skip the others. For example, if you just want DB stats, you'll configuration will look like this:
+
+[source,yaml]
+----
+- module: couchbase
+  metricsets:
+    - syncgateway
+  period: 10s
+  hosts: ["myhost:4985"]
```

Comment:
Does it make sense to separate this metricset into several metricsets instead? For example: `syncgateway_db_stats`, `syncgateway_mem_stats`, `syncgateway_global_stats` and `syncgateway_per_replication`?

Reply:
Yeah, after internal discussion we are gonna make a new entire module with. Thanks!